src/plugins/console/public/lib/autocomplete/autocomplete.ts

```diff
@@ -140,6 +140,9 @@ export function getCurrentMethodAndTokenPaths(
         continue;
       case 'punctuation.end_triple_quote':
         // reset the search for key
+        if (state === STATES.start) {
+          bodyTokenPath.unshift('"""');
+        }
```

Comment:
could we add a test for this? Also could we add a comment explaining in more details why we are doing this, I think could be very helpful for the next person who has to read this bit of code

Reply:
yes of course,no problem :)